src/Connection.php

```diff
@@ -113,6 +113,11 @@ class Connection
      */
     protected static $multiByteSupport = null;
 
+    public static $validResources = array(
+        'socket' => true,
+        'stream' => true
+    );
+
```

Comment:
@sergeyklay I was not sure if we should keep it or not. In original repository PHP 5.3 was still supported, I would be fine with supporting PHP 5.6 and newer only. What do you think?